diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -123,6 +123,23 @@ namespace diff_drive_controller
       wheel_speed_limiter_ = wheel_speed_limiter;
     }
 
+  protected:
+    /*
+     * \brief Default wheel speed limiter function
+     * \param [in, out] left_velocity  Left wheel velocity (not used)
+     * \param [in, out] right_velocity Right wheel velocity (not used)
+     */
+    virtual void wheelSpeedLimiter(double& left, double& right)
+    {
+      if (wheel_speed_limiter_)
```

Comment:
Yes, I basically mean that. I still haven't given up on syncing with upstream, so this should be cleaner if we want it accepted. :smiley: 
No rush.